packages/themes/src/tokens/button/tokens.ts

```diff
@@ -0,0 +1,109 @@
+import { Theme } from '../../types';
+import { ButtonTokenSet } from './types';
+
+export const defaultButtonTokens: ButtonTokenSet = {
```

Comment:
I don't feel it is ideal to define default tokens for a component in a separate package instead of in the component package itself. so i'm open to suggestion/discussion here

Reply:
will move the component tokens back to `react-{component}` packages

Reply:
Will that be done as part of this PR or as part of a separate one?

Reply:
I am going to do it right in this one. let me temporarily close the PR to avoid wasting people's review time!